src/core/mount.c

```diff
@@ -1133,6 +1134,12 @@ static void mount_enter_mounting(Mount *m) {
         if (r < 0)
                 goto fail;
 
+        if (STR_IN_SET(p->what, "ramfs", "tmpfs")) {
+                r = label_fix(m->where, 0);
```

Comment:
Not familiar with SELinux, but calling this here sounds racy for me. At this stage, the filesystem may not be mounted yet. Is it ok?

Reply:
Not, it didn't work.
Added into `mount_sigchld_event()`.